src/main/java/core/basesyntax/Storage.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> fruitStorage = new HashMap<>();
```

Comment:
According to the encapsulation principle, the `fruitStorage` map should not be exposed directly. Consider providing methods to manipulate the storage without giving direct access to the map.